pkg/appconsts/consensus_consts.go

```diff
@@ -3,11 +3,9 @@ package appconsts
 import "time"
 
 const (
-	TimeoutPropose = time.Second * 10
-	TimeoutCommit  = time.Second * 11
 	// GoalBlockTime is the target time interval between blocks. Since the block
 	// interval isn't enforced at consensus, the real block interval isn't
 	// guaranteed to exactly match GoalBlockTime. GoalBlockTime is currently targeted
 	// through static timeouts (i.e. TimeoutPropose, TimeoutCommit).
-	GoalBlockTime = time.Second * 15
+	GoalBlockTime = time.Second * 6
```

Comment:
Modifying this constant will impact some of the defaults in default_overrides. If we do this, there are comments that need to be updated. For example: 

https://github.com/rootulp/celestia-app/blob/17b3a727e698c298658f329362ab77d5fb045684/app/default_overrides.go#L144-L147

also we prob need to update specs with the new defaults even though the old defaults are the ones that are running on mainnet.

So we should prob reconsider modifying this constant. IMO the easiest thing to do would be to delete this constant and modify the defaults that use it to hard-coded values that match the current implementation (so based on 15 seconds).